middle/perceptual.owl

```diff
@@ -371,12 +423,21 @@ The case a) is a geometrical and mathematical, b) is geometrical and pictorial,
     
 
 
+    <!-- http://emmo.info/emmo/middle/perceptual#EMMO_bed2fe4c_dc7e_43a8_8200_6aac44030bff -->
```

Comment:
Elucidation is missing for several Symbol subclasses, like Letter, Numeral, Punctuation, Spacing and UTF8